docker-compose.yml

```diff
@@ -1,31 +1,54 @@
 version: "3.7"
 services:
   neo4j:
-    image: neo4j:3.5
+    image: neo4j:4.4.5-community
     restart: unless-stopped
     ports:
       - 7474:7474
       - 7687:7687
+    # expose:
+    #   - 7687
     volumes:
       - ./.compose/neo4j/conf:/conf
       - ./.compose/neo4j/data:/data
       - ./.compose/neo4j/import:/import
       - ./.compose/neo4j/logs:/logs
       - ./.compose/neo4j/plugins:/plugins
     environment:
-      # Raise memory limits
+      # Raise memory limits:
       - NEO4J_dbms_memory_pagecache_size=1G
       - NEO4J_dbms.memory.heap.initial_size=1G
       - NEO4J_dbms_memory_heap_max__size=1G
+      # Auth:
+      - NEO4J_AUTH=none
```

Comment:
😭 I guess I understand that this makes thing easier, especially the test suites, but I really wish we injected the credentials into neo4j, rather than turning auth off.

Reply:
@ryan-lane sure thing, I wanted it to be easier for my testing, but in a real dev/prod env we are going to inject username and password.

Reply:
Yeah, for this development setup it's probably fine, because it's also what the cartography integration tests expect. It's not ideal, but probably the right thing to do for now.

Reply:
@ryan-lane We can add in bold that it's your end user responsibility to add username and password.